tests/models.py

```diff
@@ -60,3 +60,10 @@ class Meta:
 
     def __str__(self):
         return "%s by %s" % (self.title, self.author)
+
+
+class AbtractModel(models.Model):
```

Comment:
Should we have any of the above models inherit from this?

Reply:
We could, it isn't really necessary as there are currently no cases where this would play a role (if I'm not wrong). But I will happily do it if you want.

Reply:
I agree it is fine, just wondering if you had thoughts on it!